egs/babel_multilang/s5/local/nnet3/run_common_langs.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+. ./cmd.sh
+set -e
+stage=1
+train_stage=-10
+generate_alignments=true # false if doing ctc training
+speed_perturb=true
+use_flp=false
```

Comment:
@pegahgh many of these variables are different from a normal nnet3 script. Please describe what they mean.